<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainStorm</class>
 <widget class="QDialog" name="BrainStorm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="rankButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>排位赛</string>
       </property>
      </widget>
      <widget class="QPushButton" name="singleButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>个人训练</string>
       </property>
      </widget>
      <widget class="QLabel" name="guanggao">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>281</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>281</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>430</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>查看好友</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>470</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>500</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>好友对战</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="singlepage">
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>答题</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brankButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>490</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;连续答对五道题&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>答错返回，答对继续</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="gaogao">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>211</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="singleruning">
      <widget class="QLabel" name="runing_view">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>261</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>364</y>
         <width>271</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_A">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_B">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_C">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_D">
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="gaogao1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="single_score">
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>530</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QLabel" name="singlerslut">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>241</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="singlescore">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>231</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="gaogao2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>261</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rank_wait">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>261</width>
         <height>251</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>正在等待对手</string>
       </property>
      </widget>
      <widget class="QLabel" name="gaogao3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>231</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rank_runing">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>261</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonA">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonC">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonD">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="rankview">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>241</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="selfename">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>6</pointsize>
              </font>
             </property>
             <property name="text">
              <string>小明 黄金1 ： 120</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="optname">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>6</pointsize>
              </font>
             </property>
             <property name="text">
              <string>小童 王者1 ： 300</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2"/>
        </item>
       </layout>
       <zorder>lcdNumber_2</zorder>
       <zorder>widget_3</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="rank_result">
      <widget class="QLabel" name="reslut_new">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>胜利</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="rank_brank">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>530</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QLabel" name="newRank">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>301</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>黄金1 2颗星 --&gt; 黄金1 3颗星</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="gaogao4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>490</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>添加好友</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
